euctr/crawl/base/fields.py

```diff
@@ -59,9 +62,13 @@ def __init__(self, true_value=None, **params):
         self.__true_value = true_value
 
     def parse(self, value):
-        if self.__true_value is not None:
-            value = (value.lower() == self.__true_value.lower())
-        return value
+        if value is not None:
+            if self.__true_value is not None:
+                value = (value.lower() == self.__true_value.lower())
+            return value
+        else:
+            return None
```

Comment:
Minor tactical suggestion: here, and in other cases below, test `value` against `None` first and return straight away, and then you can undent the rest of the function.  To me, this is easier to read -- it's like the code is saying `None` is special and once we've handled it, there's less for you to think about.

So the code becomes:

```
def parse(self, value):
    if value is None:
        return None

    # do stuff with value
```